xbmc/favourites/GUIDialogFavourites.cpp

```diff
@@ -37,8 +38,9 @@ using namespace XFILE;
 
 #define FAVOURITES_LIST 450
 
-CGUIDialogFavourites::CGUIDialogFavourites(void)
-    : CGUIDialog(WINDOW_DIALOG_FAVOURITES, "DialogFavourites.xml")
+CGUIDialogFavourites::CGUIDialogFavourites(void) :
```

Comment:
I personally really dislike <code>void</code> in the parameter's declaration. Is there any good reason for doing this?